### ISSUE TITLE
[DropdownMenu] Proxy focus when non-modal but portalled

### ISSUE THREAD

**Issue by benoitgrelard** [radix-ui/primitives#1308]
When non-modal, but portalled, the content is at the end of the DOM, tabbing out should proxy focus back to near the trigger.

**Comment by luisorbaiceta** [comment]
Right now, focus next is being prevented even for non-portalled menus. I'm not saying that menus should be navigated with tabs but when focus is inside a menu, tab press should close the menu and focus the next focusable element in the DOM. See this [comment](https://github.com/radix-ui/primitives/issues/1565#issuecomment-1193934920) for more reference.

**Comment by bengry** [comment]
> Hi @bengry we haven't had the chance to look at this yet.

Thanks. Any plans to take a look at this anytime soon? We're currently experimenting with a few options, and really like Radix, but it seems like this might be a deal-breaker for us (at least for the Dropdown and other floating-but-focusable components), since it doesn't meet the standard guidelines that we're required to stand in.

**Comment by pauladiniz** [comment]
Hello, do we have any updates on this?
